__tests__/app/(dashboard)/requests/getFriendRequests.test.ts

```diff
@@ -0,0 +1,97 @@
+import fetchRedis from "@/helpers/redis";
+import getFriendRequests from "@/app/(dashboard)/dashboard/requests/getFriendRequests";
+jest.mock("../../../../src/helpers/redis",()=> jest.fn());
+describe('getFriendRequests', () => {
+    beforeEach(()=>{
+        (fetchRedis as jest.Mock).mockResolvedValue([]);
+    });
+
+    afterEach(()=>{
+        jest.resetAllMocks();
+    });
+
+    test('returns an array of objects based off payloads returned from redis',async () => {
+       (fetchRedis as jest.Mock).mockImplementation(async(cmd:string, query:string)=>{
+           if (cmd ==='smembers'){
+               return['k1234','k5678', 'k90123' ]
+           }
+           const users: {[index: string]:User}  = {
```

Comment:
These tests are getting long. It would be beneficial to move the mock data or mock implementation to a separate file